presentations/scopes/index.html

```diff
@@ -101,8 +101,7 @@
   <p>
   $$
     \begin{align}
-    X &\sqsubseteq Y \quad \text{// Read as } Y \text{ satisfies } X \\
-    &\phantom{\sqsubseteq \{} \text{// Derive tighter scope-sets} \\
```

Comment:
I removed it because I needed to save a line (since I added one at the bottom) and since it didn't mean anything to me -- "derive" is a new, undefined term here, as is "tighter" and "scope-set".